Front-end/src/components/Post.jsx

```diff
@@ -158,10 +158,20 @@ const Post = (props) => {
           <img
             className="photo"
             src="https://react.semantic-ui.com/images/avatar/small/jenny.jpg"
-            onClick={() => window.location.href=`/profile-${user.id}`}
+            onClick={() => (window.location.href = `/profile-${user.id}`)}
           />
-          <p className="name" onClick={() => window.location.href=`/profile-${user.id}`}>{user.FirstName + " " + user.LastName}</p>
-          <p className="accountType" onClick={() => window.location.href=`/profile-${user.id}`}>{user.Headline}</p>
+          <p
+            className="name"
+            onClick={() => (window.location.href = `/profile-${user.id}`)}
+          >
```

Comment:
Since `/profile-${user.id}` is used in lots of places, is it possible to store them to a constant somewhere?

Reply:
I could make a function where it would do the window location change. I did not do it originally since I was worried it would make extra lines in the code file.